src/components/Tables/Table.tsx

```diff
@@ -401,6 +421,23 @@ const Table = <
 
     const rows = dataPortion.map((data, index) => ({ rowId: index, data }));
 
+    const theme = useTheme();
+
+    const dividingLineStyleOptions: DividingLineStyleOptions = {
+        dateAndSlotPrimary: {
+            colour: theme.primary.background[3],
+            thickness: "5pt",
+        },
+        dateAndSlotSecondary: {
+            colour: theme.primary.background[2],
+            thickness: "2.5pt",
+        },
+        slotPrimary: {
+            colour: theme.primary.background[3],
+            thickness: "5pt",
+        },
+    };
```

Comment:
Should we move this to `conditionalStyling.ts` and use the objects directly rather than the key?

Reply:
I agree that this bit belongs in `conditionalStyling.ts`. I can't actually include this within the `searchForBreakPoints` function (in `conditionalStyling.ts`) as this will lead to the same problem with the table and dividing lines desyncing. I have now kept most of the code similar but moved this section dealing with styling into `conditionalStyling.ts` as a function that can be called. Let me know if this is okay :)

Reply:
Ah that's annoying that it gets desynced but yeah this is ok